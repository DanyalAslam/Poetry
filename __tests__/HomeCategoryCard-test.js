/**
 * @format
 */

import 'react-native';
import React from 'react';
import HomeCategoryCard from '../src/Components/HomeCategoryCard';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';


test('renders correctly', () => {
    const tree = renderer.create(<HomeCategoryCard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
 