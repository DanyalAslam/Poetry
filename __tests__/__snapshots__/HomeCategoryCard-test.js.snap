// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "borderColor": "#f4f4f4",
      "borderRadius": 16,
      "borderWidth": 2,
      "height": 266.8,
      "marginLeft": 12,
      "width": 262.5,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": undefined,
        }
      }
      style={
        Object {
          "alignSelf": "center",
          "borderRadius": 100,
          "height": 40,
          "justifyContent": "center",
          "marginTop": 4,
          "width": 40,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignSelf": "center",
        "flex": 3,
        "justifyContent": "center",
      }
    }
  >
    <Text />
  </View>
</View>
`;
